mungegithub/mungegithub.go

```diff
@@ -128,6 +129,22 @@ func main() {
 		r.AddFlags(root, &config.Config)
 	}
 
+	// set values from config file if available.
+	viper.SetConfigName("munge-config")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath("/etc/config/")
+	viper.AddConfigPath(".")
+	if err := viper.ReadInConfig(); err != nil {
+		glog.Warningf("Cannot read configuration file: %v", err)
```

Comment:
It currently reads a config file of that name if present in one of those two locations, and sets the various option flags. If a config file is not present, it will use the flag values supplied via the commandline  

I guess we should make the configfile path an option, so that it is clearer _where_ we're reading values from. Good point.